src/global/composeRefs.ts

```diff
@@ -1,7 +1,7 @@
 import {Ref, MutableRefObject} from 'react';
 import memoizeOne from 'memoize-one';
 
-export default function composeRefs<T>(...refs: (Ref<T> | undefined)[]) {
+function composeRefs<T>(...refs: (Ref<T> | undefined)[]) {
```

Comment:
please leave this export as it may be used outside of Ring UI

Reply:
Sure, this is a valid concern.

Although I would like to propose to forbid using of this function in the ring-ui repository via ESLint, because it's easy too easy to make a mistake and use it instead of createComposedRef in render function. What do you think about it? It was the main reason why I decided to remove the export of it.

Reply:
You can deprecate it using `util-deprecate` and mark for removal in the next major version

Reply:
Thanks, I will do it in a separate PR.